grafana/README.md

```diff
@@ -0,0 +1,48 @@
+
+# Grafana Pixie Plugin Examples
+
+The following queries are for use with the [Grafana Pixie Datasource Plugin](https://grafana.com/grafana/plugins/pixie-pixie-datasource/).
+
+To learn how to install, configure, and use the datasource, check out the [tutorial](https://docs.px.dev/tutorials/integrations/grafana/).
+
+## [HTTP Errors per Service](https://github.com/pixie-io/pixie-demos/http-errors-per-service.pxl)
```

Comment:
is there a reason to put this in demos? it seems like it might be best off in 
1. the tutorial
2. the plugin repo?

Reply:
Putting them in the tutorial isn't scalable, but the plugin repo seems reasonable. Should I move it there under an `examples` folder?